pyramid_oereb/lib/renderer/extract/json_.py

```diff
@@ -8,6 +8,7 @@
 from pyramid.testing import DummyRequest
 
 from pyramid_oereb import Config, route_prefix
+from pyramid_oereb.lib import get_multiligual_element
```

Comment:
I just notice now that there is a typo in this function name (multiligual -> multilingual)

Reply:
I fixed this for all appearances.